src/main/groovy/org/ajoberstar/grgit/internal/OpSyntax.groovy

```diff
@@ -16,7 +16,6 @@
 package org.ajoberstar.grgit.internal
 
 import java.util.concurrent.Callable
-import java.util.function.Consumer
 
 import groovy.transform.PackageScope
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The [groovy.transform.PackageScope] import is never referenced [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.ajoberstar.com/coding_rules#rule_key=grvy%3Aorg.codenarc.rule.imports.UnusedImportRule)